content/docs/develop/usecases/onboard-carts-service/index.md

```diff
@@ -101,75 +105,25 @@ In case you would like to onboard your own service, please ensure that your serv
-
-    NAME                    TYPE            CLUSTER-IP      EXTERNAL-IP       PORT(S)                            AGE
-    istio-ingressgateway    LoadBalancer    10.23.245.***   ***.198.26.***    80:32399/TCP,443:31203/TCP,...     10m
-    ``` 
-
-1. Then use a browser to open Jenkins with the url `jenkins.keptn.EXTERNAL-IP.xip.io` and login using the default Jenkins credentials: `admin` / `AiTx4u8VyUV8tCKk`
-    
-    **Note:** It is highly recommended to change these credentials right after the first login.
-
-
-1. Afterwards, click on the **New Item** button and enter `carts` as name.
-
-1. Select **Multibranch Pipeline** and click on **OK**.
-    1. At *Branch Source* select *Git* and specify at *Project Repository* the github repository of your forked carts service.
-    1. At *Build Configurtion* add the extension `.ci` to the Jenkinsfile.
+## Send a new artifact and watch keptn doing the deployment 
 
-        {{< popup_image
-        link="./assets/carts_ci.png"
-        caption="CI pipeline configuration for carts">}}
-
-    1. Click Save
-
-## Build new artifact and watch keptn doing the deployment 
-
-1. Saving the Pipeline automatically starts the checkout from your Github repository and triggers the build. In case the build is not triggered, go to **carts** > **master** > **Build Now**.
-
-1. Go back to the Jenkins dashboard to see how the other pipelines get triggered automatically. In detail, after the build of the artifact (`carts` pipeline), the `deploy` pipeline is triggered for the `dev` namespace, `run_tests` is executed, before `evaluation_done` is executed. If everything goes well, the same pipelines get triggered for the `staging` and `production` namespace. In total, the pipelines will run for about 15&nbsp;minutes before you have your `carts` service deployed in your `dev`, `staging` and `production` namespace.
-
-    {{< popup_image
-      link="./assets/carts-pipeline.png"
-      caption="Successful pipeline run of the carts service">}}
-
-1. Finally, once the carts service is built, this should trigger all other pipelines that have been set up automatically. Please verify the finished pipelines:
+1. Send a new artifact for the carts service using the [keptn send new-artifact](../../reference/cli/#keptn-send-new-artifact) command.
+The used artifact is stored on Docker Hub. 
+  ```console
+  $ send new-artifact --project=sockshop --service=carts --image=docker.io/keptnexamples/carts --tag=0.8.0
+  ```
+  
+1. Go back to the Jenkins dashboard to see how the pipelines get triggered automatically. More precisely, first the `deploy` pipeline is triggered for the `dev` namespace, second the `run_tests` is executed, and finnaly, `evaluation_done` is executed. If all piplines succeed, the same pipelines get triggered for the `staging` and `production` namespace. In total, the pipelines will run for about 15&nbsp;minutes before you have your `carts` service deployed in your `dev`, `staging` and `production` namespace.
```

Comment:
I know it's nit-picking, but did we decide on using the Oxford comma or not?
... service deployed in your `dev`, `staging` and `production` namespace.
vs.
... service deployed in your `dev`, `staging`, and `production` namespace.

Reply:
We use Oxford comma.